tests/test_redact.py

```diff
@@ -12,8 +14,16 @@ def override_rule_set(data_dir):
         return build_ruleset(yaml.safe_load(rule_stream), RuleSource.OVERRIDE)
 
 
-def test_redact_svs(data_dir, tmp_path, override_rule_set):
-    redact.redact_images(data_dir / "input" / "svs", tmp_path, override_rule_set)
+@pytest.fixture(
+    params=[PurePath("svs"), PurePath("svs") / "test_svs_image_blank.svs"],
+    ids=["input_dir", "input_file"],
+)
+def svs_input_path(data_dir, request) -> Path:
```

Comment:
@manthey Note, we also switched to using a parameterized fixture to provide input for both cases, which reduced the amount of repeated code in otherwise-identical test cases.